creator-node/src/index.ts

```diff
@@ -87,26 +87,26 @@ const verifyConfigAndDb = async () => {
   }
 }
 
```

Comment:
summary of changes here? gitdiff is hard to follow

Reply:
just moving stuff into `runOneTimePreAppStartup`? or more

Reply:
yeah just moving code that's duplicated 2 or 3 times into its own function. no functional changes other than running `startAppWithoutCluster()` when cluster mode is disabled (this function is the same logic as before just without cluster listeners and forking processes)